akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -599,6 +600,17 @@ object TestSubscriber {
       self
     }
 
+    /**
+     * Fluent DSL
```

Comment:
Skip `Fluent DSL` and just make it `Java API: Assert that no message is received for the specified time. `